src/Microsoft.DotNet.GitHub.IssueLabeler/appsettings.Development.json

```diff
@@ -5,19 +5,29 @@
       "threshold": "0.4",
       "prediction_url": "",
       "can_update_labels": false,
-      "can_comment_on": false
+      "can_comment_on": false,
+      "area_owners_doc": "https://github.com/dotnet/runtime/blob/main/docs/area-owners.md",
+      "new_api_pr_label": "new-api-needs-documentation",
+      "apply_linked_issue_area_label_to_pr": true,
+      "skip_prediction": false,
+      "skip_untriaged_label": true,
+      "no_area_determined": {
+        "skip_comment": true
+      }
     },
```

Comment:
Should we add other repos here (or maybe in non development specific settings) so that we have a representation in source of every repo we want to support?  Maybe we could have defaults here that could then be overridden with Azure configuration?
IMO I prefer source for settings that we don’t expect to change often.
I noticed this because I was looking for dotnet-api-docs here for skip untriaged.

Reply:
I populated the settings with what we need from production to show the configurations.